recipe/meta.yaml

```diff
@@ -108,13 +97,9 @@ outputs:
       build:
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-        - autoconf                # [not win]
-        - automake                # [not win]
-        - libtool                 # [not win]
         - cmake
         - libgomp                 # [linux]
         - llvm-openmp             # [osx or win]
-        - autotools_clang_conda   # [win]
       host:
         - libblas
         - liblapack
```

Comment:
These have `run_exports`; you can remove them from the `run` requirements below.

Reply:
Will do